src/common/types/include/types.h

```diff
@@ -8,6 +8,8 @@
 #include <string>
 #include <vector>
 
+#include "spdlog/spdlog.h"
```

Comment:
We should try not to introduce unnecessary third party library header to `types.h`. It will propagate easily to lots of places in the codebase.

Reply:
As we discussed, i created a new logging_level_utils.h file, and put related functions here.